### ISSUE TITLE
Line height

### ISSUE THREAD

**Issue by m4rc1e** [cyrealtype/Sumana#5]
Vertical metrics are insanely loose.

![screen shot 2017-06-29 at 13 49 14](https://user-images.githubusercontent.com/7525512/27688296-0498312e-5cd2-11e7-957b-92eb87132719.png)

I suggest the following setup:

win asc: bbox ymax
win desc: bbox abs(ymin)
typo/hhea asc: 900
typo/hhea desc: -300

Then enable Use_Typo_Metrics.

I'm more than happy to do this myself.

cc @davelab6

**Comment by alexeiva** [comment]
@m4rc1e I agree, Devanagari also looks loose. Note this is a devanagari clone for Lora. 

![screen shot 2017-07-18 at 1 49 14 pm](https://user-images.githubusercontent.com/16634089/28313996-a09f829a-6bc1-11e7-9595-0d29d41d8ee3.png)

**Comment by alexeiva** [comment]
```
Highest and lowest glyphs for Sumana

Master: Regular
   Highest: Uring (top y: 961.0)
   Lowest: llVocalicMatra-deva (bottom y: -493.2)

Master: Bold
   Highest: Ring (top y: 1006.0)
   Lowest: llVocalicMatra-deva (bottom y: -517.7)
```

The metrics v metrics were set in 2000UPM scale, the font is in 1000UPM.

**Comment by alexeiva** [comment]
fonts need to be generated, but there are OT code issue to deal with first
